Bugzilla/CPAN.pm

```diff
@@ -104,6 +104,16 @@ sub feature {
     return $FEATURE_LOADED{$feature_name} = 1;
 }
 
+sub preload_features {
+    my ($class) = @_;
+    return 0 unless _CAN_HAS_FEATURE;
```

Comment:
Does it offer any material performance benefit to move this line above the my()@_ invocation?